vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -284,6 +284,12 @@ func (s *APIAggregator) AddAPIService(apiService *apiregistration.APIService) er
 	s.GenericAPIServer.Handler.NonGoRestfulMux.Handle(proxyPath, proxyHandler)
 	s.GenericAPIServer.Handler.NonGoRestfulMux.UnlistedHandlePrefix(proxyPath+"/", proxyHandler)
 
+	enableOapiHack := true
```

Comment:
What's the point of this?

Reply:
was asking @deads2k the same... this is to proxy securely to /oapi as there is no way to register that in the apiservice API.... so this is fine and we should remove this when we deprecate /oapi...

Reply:
Yeah, I get that, what I was asking why we need this true variable which is checked right in the next line?